src/components/grouped-character/grouped-character.component.tsx

```diff
@@ -90,7 +90,7 @@ export const GroupedCharacter = React.forwardRef(
       const { target } = ev;
       const { selectionEnd } = target as HTMLInputElement;
 
-      let newCursorPos = selectionEnd ?? 0;
+      let newCursorPos = selectionEnd ?? /* istanbul ignore next */ 0;
```

Comment:
**comment** (non-blocking): I'm a little uneasy at the `istanbul ignore` here. There are a lot of tests involving selection positions so I'm surprised this hasn't ended up being covered, if it presumably was before. I'd prefer to have something covering it but can live with the ignore if the test would really end up being not very relevant. I'm just curious if we know which Enzyme test was covering it that has now been removed, and if there's a sensible way to convert it.

Reply:
I'll have another look through the tests we had in Enzyme but I converted all the selectionRange tests. We did do the tests a bit different in Enzyme with simulating certain the change event and mocking the selectionRange. I'll see if I can manually trigger this line of code and write a test doing what I did manually.

Reply:
After some investigation we've found it's safe to ignore this. It was covered by [this test case](https://github.com/Sage/carbon/blob/master/src/components/grouped-character/grouped-character.spec.tsx#L91-L99) which is only testing the onChange handler, not specifically the null side of the nullish coalescing operator. It seems that it was covered by accident and was done by simulating change rather the RTL method of testing actual user behaviours.